packages/components/src/vue/components/morph-parts/principal-parts.vue

```diff
@@ -7,7 +7,7 @@
             class="alpheios-principal-parts__groupitem alpheios-principal-parts__groupitem--lemma-word"
             :lang="languageCode"
             v-if="! lemma.principalParts.includes(lemma.word)"
```

Comment:
this one is tricky and is likely going to need to be revisited with the annotation work of creating identifiers for lexical entities.  I think it's okay to include it, but probably we also need to use the displayWord when checking whether the word appears in the principalParts on line 9

Reply:
I am not quite sure about it, because I don't know the example - with numbered lemmas that has also principal parts.
So I don't know what word's variant would be there - but I believe from your notice that it won't have numbers?
I fixed according to your comment, as you are more well-known here :)